src/fun/builtins.bend

```diff
@@ -108,4 +108,9 @@ sleep hi_lo = (IO/Call IO/MAGIC "PUT_TIME" hi_lo @x (IO/Done IO/MAGIC x))
 # (defer_arg (defer_arg (defer_arg (defer @arg1 @arg2 @arg3 (f arg1 arg2 arg3)) arg1) arg2) arg3)
 defer val = @x (x val)
 defer_arg defered arg = @x (defered x arg)
-undefer defered = (defered @x x)
\ No newline at end of file
+undefer defered = (defered @x x)
+
+# log :: f24 -> f24 -> f24
+log x base = (| base x)
+# atan2 :: f24 -> f24 -> f24
```

Comment:
Explain that this means `arctan(y / x)`, with same behaviour as atan2f in the C math lib.